manuscript/100_Analysis_In_TDD.md

```diff
@@ -42,7 +42,7 @@ Gherkin
 
 Hungry? Too bad, because the Gkerkin I�am gonna talk about is not edible. It is a�notation and a�way of thinking about behaviors of the specified piece of code. It can be applied on different levels of granularity -- any behavior, whether of a�whole system or a�single class, may be described using it.
 
-We actually talked about Gherkin before in this book, just did not name it. It is the GIVEN-WHEN-THEN structure that you can see everywhere, even in code samples as comments. This time, we are stamping a�name on it and analyzing it further.
+We talked about Gherkin before in this book, just did not name it. It is the GIVEN-WHEN-THEN structure that you can see everywhere, even in code samples as comments. This time, we are stamping a�name on it and analyzing it further.
```

Comment:
I'm not aware of a specific idiom for this case.

Actually (in fact) your use of the word actually here might be ok:
"In fact we talked about the Gherkin notation before in this book, we just did not name it."

But even if the word actually *can* be used, it may be good to try to avoid it and prevent overuse.

I often use a [dictionary](http://dictionary.reference.com) and a [thesaurus](http://thesaurus.com/), or [google for phrases](https://www.google.nl/#newwindow=1&safe=off&q=affect+vs+effect) to try and find the right words.